test_profile_card_generation.py

```
@@ -0,0 +1,242 @@
+2. 터미널 2: python test_profile_card_generation.py  (테스트)
+
+주의: 실제 백엔드 API와 통신하려면 access_token이 필요합니다.
+"""
+
+import requests
+import json
+
+BASE_URL = "http://localhost:8000/api"
+
+# 실제 JWT 토큰
+TEST_ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMSIsImVtYWlsIjoic2FuZ3dvb0BuYXZlci5jb20iLCJyb2xlIjoidGFsZW50IiwiZXhwIjoxNzYwMzM1NjY5fQ.jrAoXpkLZlHip9CS5LrCscHJMk7cH1vVKU1wWf-hcLY"
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "TEST_ACCESS_TOKEN"

(S105)